.github/workflows/ci.yml

```diff
@@ -19,45 +19,41 @@ env:
   #   - https://github.com/privacy-scaling-explorations/mpz/issues/178
   # 32 seems to be big enough for the foreseeable future
   RAYON_NUM_THREADS: 32
+  # these remove the need for git in notary-server build.rs:
+  GIT_COMMIT_HASH: ${{ github.event.pull_request.head.sha || github.sha }}
+  GIT_COMMIT_TIMESTAMP: ${{ github.event.repository.updated_at}}
```

Comment:
Good catch
```suggestion
  GIT_COMMIT_TIMESTAMP: ${{ github.event.repository. pushed_at}}
```

Reply:
@heeckhau seems like `repo.pushed_at` is for commit on *any branch* — `pushed_at will be updated any time a commit is pushed to any of the repository’s branches.`, which is also wrong?